egui/src/ui.rs

```diff
@@ -338,6 +338,23 @@ impl Ui {
         self.ctx().input()
     }
 
+
+    /// The [`InputState`] of the [`Context`] associated with this [`Ui`].
+    /// Equivalent to `.ctx().input_mut()`.
+    ///
+    /// Note that this locks the [`Context`], so be careful with if-let bindings
+    /// like for [`Self::input()`].
+    /// ```
+    /// # egui::__run_test_ui(|ui| {
+    /// ui.input_mut().time = 0.0;
```

Comment:
Use the new function in the example instead (messing with time is a really bad idea)